utils/clustering-functions.R

```diff
@@ -392,9 +391,8 @@ plot_cluster_silhouette_width <- function(cluster_validity_df, num_col, point_si
     ggplot(metadata, aes(x = cluster, y = width)) +
     ggbeeswarm::geom_quasirandom(method = "pseudorandom", size = point_size) +
     geom_hline(yintercept = 0, linetype = 'dotted') +
-    labs(title = unique(metadata$cluster_type),
-         x = "Cluster Assignment") +
-    facet_wrap( ~ param_value, scale="free_x", ncol = num_col) +
+    labs(x = "Cluster Assignment") +
+    facet_wrap( ~ cluster_names_column, scale="free_x", ncol = num_col) +
```

Comment:
```suggestion
    facet_wrap( ~ cluster_names_column, scale="free_x", ncol = num_col, dir = "v" ) +
```
I think we want to add this here to so that we can have a column for each cluster type and then a row for each `nn_param` tested.